source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,748 @@
+
+At each sampling depth, 10 rarified tables are usually calculated to provide an error estimate, although this can be adjusted using the ``--p-iterations`` parameter. We can check and see if there is a relationship between the alpha diversity and metadata by passing the metadata file into the ``--m-metadata-file`` parameter.
+
+.. command-block::
+
+   qiime diversity alpha-rarefaction \
+     --i-table ./deblur_table.qza \
+     --m-metadata-file ./metadata.tsv \
+     --o-visualization ./alpha_rarefaction_curves.qzv \
+     --p-min-depth 10 \
+     --p-max-depth 2500
+
+The visualization file will give us two curves. The top curve will give the alpha diversity (observed OTUs or shannon) as a function of the sequencing depth. This is used to determine whether the richness or evenness has saturated based on the sequencing depth. The rarefaction curve should “level out” as you approach a sequencing depth. Failure to do so, especially with a diversity-only metric such as observed OTUs or Faith’s PD diversity, may indicate that the richness in the samples has not been fully saturated.
+
+The second curve shows the number of samples in each group at each sequencing depth. This is useful to determine the sampling depth where samples are lost, and whether this may be biased by metadata group. Remember that rarefaction is a two step process and samples which do not meet the rarefaction depth are filtered out of the table. So, we can use the curves to look at the number of samples by different metadata categories.
+
+If you’re still unsure whether the rarefaction depth, you can also use the sample summary to look at which samples are lost by adding metadata to the feature table summary.
+
+*Hint*: We generated this in the after we built the feature table.
+
+.. question::
+
+   Start by opening the rarefaction curves.
+
+   1. Are all metadata columns represented in the visualization? If not, which columns were excluded and why?
+   2. Which metric shows saturation and stabilization of the diversity?
+   3.  Which mouse genetic background has higher diversity, based on the curve? Which has shallower sequencing depth?
+
+   Now, let's check the feature table summary.
+
+   1. What percentage of samples are lost if we set the rarefaction depth to 1250 sequences per sample?
+   2. Which mice did the missing samples come from?
+
+**Based on the current rarefaction curve and sample summary, what sequencing depth would you pick? Why?**
+
```

Comment:
I really want a shiny box here that draws attention to the discussion. But, IDK

Reply:
Instead of using boldface text, use `.. note::` or `.. question::`. See the moving-pics tutorial for some examples of what this looks like [in the rst file](https://raw.githubusercontent.com/qiime2/docs/master/source/tutorials/moving-pictures.rst) vs. how it appears in the rendered doc... e.g., see [this section](https://docs.qiime2.org/2019.4/tutorials/moving-pictures/#alpha-rarefaction-plotting) for an example of both.

Reply:
Ive got those a couple other places. 

I want a pink box or something that highlights this. It's right below a question, so that doesn't stand out, and I tend to ignore notes, because, well, they're blue and thats probably not important. Which may just mean there are too many boxes

Reply:
`.. warning::` renders a yellow box. `.. danger::` is red. See examples [here](https://docs.qiime2.org/2019.4/data-resources/). The only problem is that this note is neither a danger nor a warning.

Reply:
It's not a note, question, warning, or danger. It's a checkpoint or a decision point. 

How hard would it be to add a new pink box for checkpoint?

Reply:
See [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html) for all possible directives. I am really not sure what would be involved in defining a new directive.

Reply:
Besides adding a new directive, it is also possible to insert vanilla HTML into the doc (see [here](https://github.com/qiime2/docs/blame/master/source/install/native.rst#L27-L72) for an example).

You could do something like:

```sphinx
.. raw:: html
   <div class="admonition" style="background:#FF78CB;">
     <strong>
       Based on the current rarefaction curve and sample summary, what
       sequencing depth would you pick? Why?
     </strong>
   </div> 
```

which would render like:

![image](https://user-images.githubusercontent.com/274668/58121710-e0f91880-7bbc-11e9-881f-fdc4b1562c80.png)

:woman_singer: :man_singer:

Reply:
I just made a new directive, but the vanilla HTML might have been better. Pro either way is that now I know more about Sphinx directives?

<img width="806" alt="3" src="https://user-images.githubusercontent.com/5534846/58124155-d1ee8680-7c0d-11e9-992c-54c4b22d6d15.png">